### DISCUSSION TITLE
Replace libaom To libdav1d for better av1 accelerations

### DISCUSSION THREAD

**Discussion by relvinarsenio** [linuxdeepin/developer-center#3396] [Features Request & Ideas | 特性请求 & 头脑风暴]
av1 codecs has no hardware accelerations 
Please Update ffmpeg and enable options libdav1d and disable libaom
Libdav1d better performance than libaom for av1 codecs 
And use dav1d library instead of libaom for deepin movie
The problem is the performance of av1 and h264 is very bad no accelerations on my laptop.

and performance features in the control center have no effect on system performance.
unlike gnome which has similar features, but the features in gnome, affect system performance.

I hope deepin changes base from debian to fedora or another linux distribution, which has a newer package version.

**Comment by BLumia** [comment]
cc @Zeno-sole

**Comment by Zeno-sole** [comment]
We will conduct relevant assessments

**Reply by BLumia** [reply]
cc @lzwind @wyu71 from [@deepin-community/sig-deepin-media](https://github.com/deepin-community/SIG/blob/master/sig/deepin-media/MEMBERS.md)
